app/views/devise/confirmations/new.html.erb

```diff
@@ -1,16 +1,18 @@
-<h2>Resend confirmation instructions</h2>
-
-<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %> 
+<div class="form-container">
+<%= devise_error_messages! %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h2>Resend confirmation instructions</h2>
+      <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
```

Comment:
seems like this might fit better immediately inside the `<div class="panel-body">` tag below